dataactvalidator/config/sqlrules/a1_appropriations.sql

```diff
@@ -0,0 +1,16 @@
+SELECT
+    row_number,
+    allocation_transfer_agency,
+    agency_identifier,
+    beginning_period_of_availa,
+    ending_period_of_availabil,
+    availability_type_code,
+    main_account_code,
+    sub_account_code
+FROM appropriation as approp
+WHERE submission_id IS NOT DISTINCT FROM {}
```

Comment:
Whoops, got a bit carried away with replace all, will fix that after my tests on a different branch finish running.